src/main/java/org/fcrepo/camel/FcrepoResponse.java

```diff
@@ -46,7 +47,7 @@
      * @param body the response body stream
      */
     public FcrepoResponse(final URI url, final int statusCode,
-            final String contentType, final URI location, final InputStream body) {
+            final Optional<String> contentType, final Optional<URI> location, final InputStream body) {
         this.setUrl(url);
```

Comment:
@ajs6f some questions on this change -- the `FcrepoResponse` class has a number of `String` fields, two of which were changed to `Optional<String>`. Is this completely unnecessary (as per the comment above)? Should they be left as plain `String`s with null checks in the implementation code?